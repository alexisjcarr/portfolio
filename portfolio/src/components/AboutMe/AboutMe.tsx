import * as React from 'react'

import './AboutMe.scss'
import me from '../../assets/images/me.jpg'

const AboutMe: React.SFC = () => {
  return (
    <div className="container" id="aboutme">
      <div className="card">
        <h1>about me</h1>
        <hr />
        <img src={me} alt="profile pic" className="profile-pic" />
        <br />
        <br />
        <h1>{`<alexis j. carr/>`}</h1>
        <br />
        <h3>
          software engineer @ github | clarinetist | writer
        </h3>
        <br />
        <h3>a perpetual work in progress. i really like turtles.</h3>
        <br />
        <h4>
          University of Houston '16 ã€‹Texas A&M University '19
        </h4>
      </div>
    </div>
  )
}

export default AboutMe
