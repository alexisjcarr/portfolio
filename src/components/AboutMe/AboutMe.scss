.container {
  display: flex;
  justify-content: center;
  align-items: center;

  .card {
    background-color: black;
    width: 90%;
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
    text-align: center;
    margin-top: 100px;
    color: greenyellow;

    h3 {
      line-height: 1.6;
    }

    h4 {
      line-height: 1.6;
    }

    @media (max-width: 500px) {
      h1 {
        font-size: 20px;
      }

      h3 {
        font-size: 12px;
        line-height: 1.6;
      }

      h4 {
        font-size: 10px;
        line-height: 1.6;
      }
    }

    .profile-pic {
      height: 20%;
      width: 20%;
      border-radius: 1000px;
      border: 5px solid yellowgreen;
      margin-top: 20px;

      @media (max-width: 500px) {
        height: 50%;
        width: 50%;
      }
    }
  }
}
