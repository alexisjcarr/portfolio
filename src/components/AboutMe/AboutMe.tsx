import * as React from "react";

import "./AboutMe.scss";
import me from "../../assets/images/me.jpg";

const AboutMe: React.SFC = () => {
  return (
    <div className="container" id="aboutme">
      <div className="card">
        <h1>about me</h1>
        <img src={me} alt="profile pic" className="profile-pic" />
        <br />
        <br />
        <h1>{`<alexis j. carr/>`}</h1>
        <br />
        <h3>
          fullstack software engineer | biomedical engineer | slam poet | writer
        </h3>
        <br />
        <h3>a perpetual work in progress. i really like turtles.</h3>
        <br />
        <h4>
          University of Houston '16 》Texas A&M University '19 》 Lambda School
          '20
        </h4>
      </div>
    </div>
  );
};

export default AboutMe;
