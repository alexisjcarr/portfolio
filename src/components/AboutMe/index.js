export { default } from "./AboutMe.tsx";
