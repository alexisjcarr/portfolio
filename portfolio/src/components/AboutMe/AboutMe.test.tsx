import React from 'react'
import { render, cleanup } from '@testing-library/react'

import AboutMe from './AboutMe'

describe('<AboutMe />', () => {
  let originalWarning, originalError

  beforeEach(() => {
    originalWarning = console.warn
    originalError = console.error
    console.error = jest.fn()
    console.warn = jest.fn() // temporarily sets console.warn to a jest function
  })

  afterEach(() => {
    cleanup()
    console.warn = originalWarning // turns it back into the original error after test runs
    console.error = originalError
  })

  it('matches snapshot', () => {
    expect(render(<AboutMe />)).toMatchSnapshot()
  })
})
