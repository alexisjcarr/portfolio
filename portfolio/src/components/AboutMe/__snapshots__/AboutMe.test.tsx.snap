// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutMe /> matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
        id="aboutme"
      >
        <div
          class="card"
        >
          <h1>
            about me
          </h1>
          <img
            alt="profile pic"
            class="profile-pic"
            src="me.jpg"
          />
          <br />
          <br />
          <h1>
            &lt;alexis j. carr/&gt;
          </h1>
          <br />
          <h3>
            fullstack software engineer | biomedical engineer | slam poet | writer
          </h3>
          <br />
          <h3>
            a perpetual work in progress. i really like turtles.
          </h3>
          <br />
          <h4>
            University of Houston '16 》Texas A&M University '19 》 Lambda School '20
          </h4>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
      id="aboutme"
    >
      <div
        class="card"
      >
        <h1>
          about me
        </h1>
        <img
          alt="profile pic"
          class="profile-pic"
          src="me.jpg"
        />
        <br />
        <br />
        <h1>
          &lt;alexis j. carr/&gt;
        </h1>
        <br />
        <h3>
          fullstack software engineer | biomedical engineer | slam poet | writer
        </h3>
        <br />
        <h3>
          a perpetual work in progress. i really like turtles.
        </h3>
        <br />
        <h4>
          University of Houston '16 》Texas A&M University '19 》 Lambda School '20
        </h4>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
